### PR TITLE
[ALS-6924] Take a Tour for PIC-SURE Demo

### PR DESCRIPTION
**Pull request by Gcolon021**
[ALS-6924] Take a Tour for PIC-SURE Demo
[ALS-7795] Tour on Discover does not apply a filter

### CODE REVIEW

File: src/routes/(picsure)/(public)/discover/+page.svelte [hunk 1]
```diff
@@ -2,7 +2,16 @@
   import Content from '$lib/components/Content.svelte';
   import { branding } from '$lib/configuration';
   import Explorer from '$lib/components/explorer/Explorer.svelte';
-  import openTour from '$lib/assets/openTourConfiguration.json';
+  import TourData from '$lib/assets/TourConfiguration.json';
+
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (JamesPeck) [inline src/routes/(picsure)/(public)/discover/+page.svelte:7]:
```suggestion
  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```

File: src/routes/(picsure)/(authorized)/explorer/+page.svelte [hunk 2]
```diff
@@ -2,7 +2,17 @@
   import { branding } from '$lib/configuration';
   import Content from '$lib/components/Content.svelte';
   import Explorer from '$lib/components/explorer/Explorer.svelte';
-  import authTour from '$lib/assets/authTourConfiguration.json';
+  import TourData from '$lib/assets/TourConfiguration.json';
+
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (JamesPeck) [inline src/routes/(picsure)/(authorized)/explorer/+page.svelte:7]:
```suggestion
  /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
```

File: src/routes/(picsure)/(authorized)/explorer/+page.svelte [hunk 3]
```diff
@@ -2,7 +2,17 @@
...
+  const Tour: Record<string, any> = TourData;
+
+  const tourName: string | undefined = import.meta.env.VITE_AUTH_TOUR_NAME;
```
Comment (JamesPeck) [inline src/routes/(picsure)/(authorized)/explorer/+page.svelte:10]: Do you mind adding this to the .env example file?

File: src/routes/(picsure)/(public)/discover/+page.svelte [hunk 4]
```diff
@@ -2,7 +2,16 @@
...
+  const Tour: Record<string, any> = TourData;
+  const tourName = import.meta.env.VITE_OPEN_TOUR_NAME;
```
Comment (JamesPeck) [inline src/routes/(picsure)/(public)/discover/+page.svelte:9]: Do you mind adding this to the .env example file?

**Review by JamesPeck — CHANGES_REQUESTED** [review]

**Comment by Gcolon021** [comment]
@JamesPeck I am re-running the test, but I am not hopeful.

Here are my local test passing.
image

**Review by JamesPeck — APPROVED** [review]

**Pull request merged**